### DISCUSSION TITLE
关于mmkv未存value类型的疑问

### DISCUSSION THREAD

**Discussion by ghost** [Tencent/MMKV#846] [Ideas]
https://github.com/Tencent/MMKV/issues/168
这里说mmkv之所以不存value的类型是因为会增加space占用，但linux的页大小4KB，其实会多出不少空间，如果像FastKV
https://juejin.cn/post/7018522454171582500
这样用6bits存储value类型，貌似也不会多占多少空间，读取存储耗时也不多，getAll()也能实现。

不知道不存value类型是否是作者经过大数据测试后的结论，或者以后的版本会考虑这种方案？

**Comment by lingol** [comment]
NO, we will never add this feature. If you really need it you can use whatever suits you best.

**Reply by ghost** [reply]
got it.
